crates/client/src/config.rs

```diff
@@ -94,8 +209,80 @@ impl Config {
         }
         throw!(Error::BadWorkspace)
     }
+    pub fn get_env_variable(self, cli_input: String) -> String {
```

Comment:
The name `get_env_variable` is not accurate, I think `get_fuzz_args` or something like that would be better...